### PR TITLE
aegisub.md updates

### PR DESCRIPTION
**Pull request by OrangeChannel**
Notable changes
- d9657ab updated pictures and related text to line0's recent Aegisub build
- 341a0b2 trying something new here, which could be useful in the future
   - 0d4795a currently the only way to get this to work, however, and it's pretty bad

New features
- 1bc64ca uses the Dependency Control glossary entry to avoid redundancy and un-related information
   - mostly as a proof of concept here, but could be used a lot in other articles

### CODE REVIEW

File: typesetting/aegisub.md [hunk 1]
```diff
@@ -61,105 +61,115 @@ Here is a list of tools you will want to download:
...
+and how these two are different,
+you can read up [here](../archived-websites/bt601-vs-bt709.md),
+but the gist of it is this:
+BT.601 is for Standard Definition video
+and BT.709 is for High Definition video[^3].
+
+Manually setting the script to BT.601 could
+*irreversibly ruin the colors of any typesetting,
+dialogue,
+or kfx already in the script*.
```
Comment (FichteFoll) [inline typesetting/aegisub.md]: I guess we can make this bold.

File: typesetting/aegisub.md [hunk 2]
```diff
@@ -61,105 +61,115 @@ Here is a list of tools you will want to download:
...
+will automatically be set to match the video source.
+
+![Aegisub 8975-master-8d77da3 script properties 1](images/script_properties-1.png)
+
+For most cases with modern fansubbing,
+the BT.709 color space will be used
+as opposed to the legacy BT.601 color space.
+If you want a more in-depth explanation of color matrices
+and how these two are different,
+you can read up [here](../archived-websites/bt601-vs-bt709.md),
```
Comment (FichteFoll) [inline typesetting/aegisub.md]: See my comment on using "here" as hyperlink text in the other PR.

File: typesetting/aegisub.md [hunk 3]
```diff
@@ -61,105 +61,115 @@ Here is a list of tools you will want to download:
...
+
+Manually setting the script to BT.601 could
+*irreversibly ruin the colors of any typesetting,
+dialogue,
+or kfx already in the script*.
+Even worse,
+some video renderers will read this setting from the muxed subtitles
+and render the video to match it.
+
+
```
Comment (FichteFoll) [inline typesetting/aegisub.md]: Two line breaks.

File: typesetting/aegisub.md [hunk 4]
```diff
@@ -61,105 +61,115 @@ Here is a list of tools you will want to download:
...
+The recommended build of Aegisub comes pre-equipped with libass,
+so no manual settings change is needed.
+The following is a brief history of subtitle renderers.
+
+Just a few years ago,
+there was a pretty clear consensus on which
+subtitle renderer to use for anime and softsubs.
+These days, not so much.
+It used to be that [VSFilter][] was the best renderer around
```
Comment (FichteFoll) [inline typesetting/aegisub.md]: Could also say it was the only or "original" renderer. libass surfaced much later and xy is a fork afaik.

File: typesetting/aegisub.md [hunk 5]
```diff
@@ -61,105 +61,115 @@ Here is a list of tools you will want to download:
...
+and was the only supported renderer by most fansub groups.
+However, it was eventually replaced with [xy-vsfilter][]
+and then later replaced with [xySubFilter][]
+because VSFilter and xy-vsfilter were not keeping up
+with the demands of subtitles.
```
Comment (FichteFoll) [inline typesetting/aegisub.md]: Probably noteworthy that it's about the *resource requirements* of subtitles or rather that the performance of renderers could not keep up. Functionally they are still the same.

File: typesetting/aegisub.md [hunk 6]
```diff
@@ -61,105 +61,115 @@ Here is a list of tools you will want to download:
...
+both in speed and compatibility with advanced subtitling
+in part due to contributions from members of the fansub community.
+At the end of the day,
+which renderer you choose is up to you,
+but we recommend libass.
+It is maintained,
+cross-platform,
+able to handle most typesetting,
+and has been integrated into many commercial
+and open-source software products.
```
Comment (FichteFoll) [inline typesetting/aegisub.md:164]: Maybe add a sentence on which renderer is used where (media player)? VSFilter and forks only work on Windows, for example.

File: typesetting/aegisub.md [hunk 7]
```diff
@@ -61,105 +61,115 @@ Here is a list of tools you will want to download:
...
 
-## Configuring Aegisub
 
-For now, just change your settings to reflect the following. If you've
-made any changes previously for another fansub role, be careful not to
-overwrite those. When in doubt, ask someone with Aegisub experience.
-Settings can be accessed via *View \> Options* or with the hotkey *Alt +
-O*.
-
-![aegisub64\_2017-09-08\_01-50-08.png](images/cnvimage100.png)
```
Comment (FichteFoll) [inline typesetting/aegisub.md:60]: Don't you want to delete these images? Their deletion isn't included in the commit (or GitHub is lying to me).

File: typesetting/aegisub.md [hunk 8]
```diff
@@ -31,14 +31,13 @@ Here is a list of tools you will want to download:
       - Option 1: [Mocha Pro standalone app][]
         - Look for it on *certain websites*.
         - On Windows, Mocha **requires** [Quicktime][] to be installed.
-          More information can be found [here][quicktimeFAQ]
+          More information can be found [here][quicktimeFAQ].
       - Option 2: [Blender][][^1]
         - Requires a plug-in that will be discussed in a later section.
   - [x264 binary][][^2]
       - Download the latest **8-bit** binary for your platform.
```
Comment (FichteFoll) [inline typesetting/aegisub.md]: Just call it the "normal" binary imo.

**Review by FichteFoll — CHANGES_REQUESTED** [review]

File: typesetting/aegisub.md (hunk 7 above)
Comment (OrangeChannel) [reply typesetting/aegisub.md:60]: Yes, didn’t know how to do it properly without having merging issues.

Comment (FichteFoll) [reply typesetting/aegisub.md:60]: There shouldn't be any merge issues if you aren't modfiying the same file/location in a text file in a different commit on a different branch.

Comment (OrangeChannel) [reply typesetting/aegisub.md:60]: Oh, okay, I'll delete them in a new commit on this PR.

**Review by FichteFoll — APPROVED** [review]

**Pull request merged**